tests/embed_tests/test_local_inference.py

```
@@ -30,28 +33,48 @@
+
+@pytest.fixture(scope="module", autouse=True)
+def cleanup():
+    yield
+    shutil.rmtree(SESSION_TMP_DIR)
+
 
 @lru_cache
 def _cached_text_embedding(model_name, *args, **kwargs):
+    kwargs["cache_dir"] = kwargs["cache_dir"] or _get_cache_dir_name(model_name)
+    kwargs["local_files_only"] = kwargs.get("local_files_only", Path(kwargs["cache_dir"]).exists())
     return TextEmbedding(model_name=model_name, *args, **kwargs)
```

Ruff (0.14.1)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)